README.md

```diff
@@ -100,6 +100,27 @@ const LIMIT = 10;
 $queryBuilder = QueryBuilder::createNew(0, 10, 0.3);
 ```
 
+#### GeoDistance query
+Add property on the index :
+```json
+{
+//...
+    "_source": {
+    //...
+        "field_name" : {
```

Comment:
doesn't it need the type ? or is it how it looks in ES ?

Reply:
is it how it looks in ES